packages/grafana-data/src/datetime/formatter.test.ts

```diff
@@ -72,5 +72,91 @@ describe('dateTimeFormat', () => {
     it('should format with default formatting in correct time zone', () => {
       expect(dateTimeFormat(1587126975779, options)).toBe('2020-04-17 15:36:15');
     });
+    it('should format with default formatting in correct time zone', () => {
+      const options = { timeZone: 'Europe/London' };
+      expect(dateTimeFormat(1587126975779, options)).toBe('2020-04-17 13:36:15');
+    });
+    it('should format with default formatting in correct time zone', () => {
+      const options = { timeZone: 'Europe/Berlin' };
+      expect(dateTimeFormat(1587126975779, options)).toBe('2020-04-17 14:36:15');
+    });
+    it('should format with default formatting in correct time zone', () => {
+      const options = { timeZone: 'Europe/Moscow' };
+      expect(dateTimeFormat(1587126975779, options)).toBe('2020-04-17 15:36:15');
+    });
+    it('should format with default formatting in correct time zone', () => {
+      const options = { timeZone: 'Europe/Madrid' };
+      expect(dateTimeFormat(1587126975779, options)).toBe('2020-04-17 14:36:15');
+    });
+    it('should format with default formatting in correct time zone', () => {
+      const options = { timeZone: 'America/New_York' };
+      expect(dateTimeFormat(1587126975779, options)).toBe('2020-04-17 08:36:15');
+    });
+    it('should format with default formatting in correct time zone', () => {
+      const options = { timeZone: 'America/Chicago' };
+      expect(dateTimeFormat(1587126975779, options)).toBe('2020-04-17 07:36:15');
+    });
+    it('should format with default formatting in correct time zone', () => {
+      const options = { timeZone: 'America/Denver' };
+      expect(dateTimeFormat(1587126975779, options)).toBe('2020-04-17 06:36:15');
+    });
+  });
```

Comment:
This is a great use case for a parametrized test! https://jestjs.io/docs/api#testeachtablename-fn-timeout

You  just create a table with the input and expected output, which reduces quite a lot of code duplication here.